src/config/app.ts

```diff
@@ -1,15 +1,19 @@
-import express from "express";
+import express, { Application, Request, Response } from "express";
 import bodyParser from "body-parser";
 import dotenv from "dotenv";
 import morgan from "morgan";
+import passport from "passport";
 import cors from "cors";
+import session from "express-session";
 import serverRoute from "../routes/index";
 import { pool } from "./database";
 
 dotenv.config();
 
+const secret: string | undefined = process.env.SESSION_SECRET;
```

Comment:
Why did you put this here?

I feel like it would be more appreciable in the class as it is being used there.

You can add it as a private property on `line 17` for example:

```typescript
private readonly secret?: string = process.env.SESSION_SECRET;
```